src/style.css

```
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 100px;
+  height: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars > * {
+  width: 13px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars :nth-child(n) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--0 :nth-child(n) {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 :nth-child(n+2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 :nth-child(n+3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 :nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 :nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
```

You can combine it like this: 
```suggestion
.stars--0 :nth-child(n),
.stars--1 :nth-child(n+2),
.stars--2 :nth-child(n+3),
.stars--3 :nth-child(n+4),
.stars--4 :nth-child(n+5) {
  background-image: url("images/star.svg");
}
```